tests/test_notify.py

```diff
@@ -57,6 +58,7 @@ def test_notify_address_from(self):
                     'queue': queue_url}}]
             }, session_factory=session_factory)
         resources = policy.run()
+        self.assertEqual(policy.data.get('actions')[0].get('to'), ['to@example.com'])
```

Comment:
this doesn't really assert the deepcopy afaics

Reply:
without deepcopy, `to` would evaluate as `['to@example.com','me@example.com']`